scripts/systNtuple.py

```diff
@@ -102,12 +99,9 @@
     if os.path.exists (jobsDir) : os.system ('rm -f ' + jobsDir + '/*')
     else                        : os.system ('mkdir -p ' + jobsDir)
 
-    n = int (0)
-    if opt.nI >=0:
-        n = int (opt.nI)
     commandFile = open (jobsDir + '/submit.sh', 'w')
     for inputfile in inputfiles : 
-
+	n = int(inputfile.split('/')[-1][7:-5])
```

Comment:
is this an indentation error?

Reply:
Never mix spaces and tabs... Should be fine now